### PR TITLE
chore(lidar_centerpoint): add a check between the model and config's class size

### PR DESCRIPTION
**Pull request by knzo25**
## Description
add a check between the model and config's class size

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: perception/lidar_centerpoint/lib/network/network_trt.cpp
Comment (yukke42) [inline perception/lidar_centerpoint/lib/network/network_trt.cpp:68]: Cloud you try to use `logger_`? It's a wapper of ros message for tensort library.
https://github.com/autowarefoundation/autoware.universe/blob/main/common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp#L38-L73

Comment (knzo25) [reply perception/lidar_centerpoint/lib/network/network_trt.cpp:68]: @yukke42 
Currently the caller also uses `std::cout`
`perception/lidar_centerpoint/lib/network/tensorrt_wrapper.cpp`

Should I replace all `std::cout` for consistency?

Comment (yukke42) [reply perception/lidar_centerpoint/lib/network/network_trt.cpp:68]: @knzo25 I forget to replace them in the previous pr..., Cloud you replace others as well? :pray:

Comment (knzo25) [reply perception/lidar_centerpoint/lib/network/network_trt.cpp]: Replaced the `std::cout` calls in [f0f0d38](https://github.com/autowarefoundation/autoware.universe/pull/3101/commits/f0f0d380370824f731d20277a5df5511f67d4d39)
Also fixed some typos

**Review by yukke42 — APPROVED** [review]
LGTM
It works fine!

**Comment by knzo25** [comment]
?
Do we need the approval from all the reviewers?

**Comment by yukke42** [comment]
Either @wep21 or @manato. They are code owners of the `tensorrt_common` package.

**Comment by knzo25** [comment]
Ooh right, thanks !

File: common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp
Comment (wep21) [inline common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp:71]: This logger is passed to TensorRT API, so you should not use it to output log. You can call rclcpp logger macro directly.

Comment (knzo25) [reply common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp:71]: I understand, but was asked to use it this way:

> Cloud you try to use `logger_`? It's a wapper of ros message for tensort library. https://github.com/autowarefoundation/autoware.universe/blob/main/common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp#L38-L73

Comment (knzo25) [reply common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp:71]: In the same way, I did not use RCLCPP from the beginning since the caller of the method I was using, was already using `std::cout` (-> code consistency). 
Should I just RCLCPP in all the changes in this commit  @yukke42

Comment (yukke42) [reply common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp:71]: @knzo25 Cloud you follow @wep21's proposal? :bow:

**Comment by knzo25** [comment]
@yukke42 
Sorry for the delay. I applied the changes

File: perception/lidar_centerpoint/lib/network/tensorrt_wrapper.cpp
Comment (yukke42) [inline perception/lidar_centerpoint/lib/network/tensorrt_wrapper.cpp:71]:
```suggestion
    RCLCPP_ERROR(rclcpp::get_logger("lidar_centerpoint"), "Failed to create context");
```

**Review by yukke42 — APPROVED** [review]
LGTM

**Pull request merged**
